openpype/settings/constants.py

```diff
@@ -2,14 +2,14 @@
 
 
 # Metadata keys for work with studio and project overrides
-M_OVERRIDEN_KEY = "__overriden_keys__"
+M_OVERRIDDEN_KEY = "__overridden_keys__"
```

Comment:
This is too dangerous for this PR which will break all existing settings.
```suggestion
M_OVERRIDDEN_KEY = "__overriden_keys__"
```